tests/test_ERC721.py

```diff
@@ -0,0 +1,218 @@
+import pytest
+import asyncio
+from starkware.starknet.testing.starknet import Starknet
+from utils import Signer, uint, str_to_felt
+
+owner_signer = Signer(123456789987654321)
+friend_signer = Signer(69420)
+
+
+@pytest.fixture(scope='module')
+def event_loop():
+    return asyncio.new_event_loop()
+
+
+async def ownable_factory():
+    starknet = await Starknet.empty()
+    owner = await starknet.deploy(
+        "contracts/utils/Account.cairo",
+        constructor_calldata=[owner_signer.public_key]
+    )
+
+    friend = await starknet.deploy(
+        "contracts/utils/Account.cairo",
+        constructor_calldata=[friend_signer.public_key]
+    )
+
+    erc721 = await starknet.deploy(
+        "tests/mocks/MockERC721.cairo",
+        constructor_calldata=[
+            str_to_felt("Test Contract"),
+            str_to_felt("TEST")
+        ]
+    )
+    return starknet, erc721, owner, friend
+
+
+@pytest.mark.asyncio
+async def test_constructor():
+    _, erc721, _, _ = await ownable_factory()
+    expected_name = await erc721.name().call()
+    assert expected_name.result.name == str_to_felt("Test Contract")
+    expected_symbol = await erc721.symbol().call()
+    assert expected_symbol.result.symbol == str_to_felt("TEST")
+
+
+@pytest.mark.asyncio
+async def test_mint():
+    _, erc721, owner, _ = await ownable_factory()
+    token = uint(0)
```

Comment:
i would add some random # here (69420) to make sure we mint the right token id (not just always 0)